tests/unit/amo/components/TestSearchResult.js

```diff
@@ -144,6 +144,50 @@ describe(__filename, () => {
     expect(root).toHaveClassName('SearchResult--theme');
   });
 
+  it('displays the thumbnail image as the default src for static theme', () => {
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      type: ADDON_TYPE_STATIC_THEME,
+    });
+    const root = render({ addon });
+    const image = root.find('.SearchResult-icon');
+
+    expect(image.prop('src')).toEqual(
+      'https://addons.cdn.mozilla.net/4444/image.png',
```

Comment:
We generally prefer to set data in our test that we're going to assert about, so rather than checking that this has the value from `fakeAddon`, you should create an add-on in the test, using `fakeAddon` as a basis, but providing your own value for `thumbnail_url`.

Reply:
I think I got it...I'll update...thanks : )